package.json

```diff
@@ -38,22 +38,23 @@
     "add": "^2.0.6",
     "axios": "^0.21.0",
     "axios-mock-adapter": "^1.19.0",
-    "babel-plugin-module-resolver": "^4.0.0",
     "json-stable-stringify": "^1.0.1",
+    "babel-plugin-module-resolver": "^4.1.0",
+    "jest-styled-components": "^7.0.3",
     "jwt-decode": "^3.1.2",
     "metro-react-native-babel-preset": "^0.66.2",
     "moment": "^2.29.1",
     "prop-types": "^15.7.2",
     "punycode": "^2.1.1",
     "react": "16.13.1",
-    "react-native": "0.64.1",
+    "react-native": "0.64.2",
```

Comment:
I think the latest stable version of react-native is 0.66.2. So it might be a good idea to bump to that version instead?

Reply:
Cool. Will try it out. The reason to upgrade in small steps is to not break something :)

Reply:
I agree with you. But someday we might need to do that big jump in version number, and the sooner the better

Reply:
Will keep